README.md

```diff
@@ -13,12 +13,13 @@ written in [Rust](http://rustlang.org/).
 The Test-API is available under `http://dev.ofdb.io/v0/`.
 
 The current API is documented within the [openapi.yaml](https://github.com/kartevonmorgen/openfairdb/blob/master/openapi.yaml) file.
-You can render the API documentation e.g. with the swagger editor:
+You can render the API documentation e.g. with the [swagger editor](https://app.swaggerhub.com/apis/Kartevonmorgen/open-fair_db_api):
```

Comment:
Why has this inconsistent naming "open-fair_db" vs. "openfairdb" been introduced? 

@flosse Is https://app.swaggerhub.com/apis/Kartevonmorgen/open-fair_db_api maintained by us? Otherwise it should not be referred to in the README.

Reply:
I would also prefer that it is always written openfairdb

Reply:
I was able to change the slug of the swagger URL: https://app.swaggerhub.com/apis/Kartevonmorgen/openfairdb/0.10.3

As soon as there are changes in the openapi.yaml i can check, if swagger can be updated with one click...